pkg/tar/tar_test.go

```diff
@@ -236,6 +278,18 @@ func verifyDirectory(t *testing.T, dir string, files []fileDesc) {
 					t.Errorf("File content is not equal for %s. Expected: %s, Actual: %s\n",
 						relpath, fd.content, content)
 				}
+				if isSymLink(fd.mode) {
+					target, err := os.Readlink(path)
+					if err != nil {
+						return err
+					}
+					if target != fd.target {
+						t.Errorf(
+							"Symbolic link %q points to wrong path. Expected: %s, Actual: %s\n",
+							fd.name, fd.target, target,
+						)
```

Comment:
I have vim window set to 80 characters, long lines are annoying for me :) Uber-idiomatic would be to create special error and pass these three as arguments, but that is too much for this.